docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,546 @@
+
+### Global class names
+
+In v3, we heard how frustrating using the `classes` prop API correctly can sometimes be.
+In v4, we made [a step](/blog/material-ui-v4-is-out/#customization) toward adding global class names.
+They are present, as long as no more than one ThemeProvider is used.
+
+v5 double down on this direction by always adding global class names on the host DOM nodes.
+These class names are available for customizing the child elements.
+It can simplify the customization of complex components.
+
+For instance, compare these tree options to turn the input outlined border color red:
+
+```tsx
+import TextField from '@mui/material/TextField';
+import { outlinedInputClasses } from '@mui/material/OutlinedInput';
+import { styled } from '@mui/material/styles';
+
+// Option 1: global class
+const CustomizedTextField1 = styled(TextField)({
+  '& .MuiOutlinedInput-notchedOutline': {
+    borderColor: 'red',
+  },
+});
+
+// Option 2: global class + const
+const CustomizedTextField2 = styled(TextField)({
+  [`& .${outlinedInputClasses.notchedOutline}`]: {
+    borderColor: 'red',
+  },
+});
+
+// Option 3: classes prop (before)
+const CustomizedTextField3 = styled((props) => (
+  <TextField
+    {...props}
+    variant="outlined"
+    InputProps={{ classes: { notchedOutline: 'foo' } }}
+  />
+))({
+  '& .foo': {
+    borderColor: 'red',
+  },
+}) as typeof TextField;
+```
+
+<p class="blog-description"><a href="https://codesandbox.io/s/zealous-dawn-0yr4g?file=/src/App.tsx">Codesandbox</a></p>
+
+Option 1 is the simplest but if you want more type safety and do not use a magic string (`MuiOutlinedInput-notchedOutline`), you can use Option 2.
+
+### Base components (alpha)
+
+While hooks were high-risk experimentation when React released them in 2018, they are now ubiquitous.
+This is a great opportunity for MUI to expose more flexibility: headless components.
+
+A key reason why developers pick MUI is to be able to build UIs faster.
+When they depend on us, they make a tradeoff.
+They estimate that applying new styles on top of the Material Design components will be faster than creating components from scratch or picking another library.
+They estimate that it will be performant enough, and they won't miss too much freedom.
+
+This tradeoff work **really well** when having a small, constrained engineering team or a large team building internal (/secondary) tools.
+But what about the medium/large size engineering team that works on ambitious projects? Shouldn't they have a better option than building the components from scratch to not include Material Design and maximize freedom?
+
+This is this problem that our team started working on.
+We are isolating the logic of the Material Design components into hooks and unstyled components.
+While the effort is still in alpha, you can already find the first building blocks in the `@mui/base` package.
```

Comment:
> `@mui/base`

![I see what you did there](https://media.giphy.com/media/l46CxpYSFoFtN11ug/giphy.gif)

Reply:
At least it's still in alpha we can change it 🤫

Reply:
Haha, it's the only way I could find to write this section without being confused about the name: https://github.com/mui-org/material-ui/discussions/28271#discussioncomment-1310705.

Reply:
@mnajdova @michaldudak Let me know how you would like to handle this one. I think that the first question is about the header. Is `### Base components (alpha)` OK? Or should it be `### Core components`?

Reply:
If there is a consensus about calling the package `@mui/base` (I'm ok with it), we can title it `Base components (alpha)`. Another safe option would be `Unstyled components (alpha)`, even if it doesn't match the package name. `Core` is now too overloaded.

If we go with `@mui/base`, we should rename the package as soon as possible, to there's no mismatch between the post and reality.

Reply:
+1 for `Unstyled components (alpha)` - it's descriptive and it doesn't need to match the package name.


> If we go with @mui/base, we should rename the package as soon as possible, to there's no mismatch between the post and reality.

Agree 👍

Reply:
I'm in with whatever you chose, be it Base or Unstyled. Agree that going with Core now is not a good option 😅

Reply:
I personally fail to see how `@mui/core` can work when the how the product is named MUI Core. I think that most developers would assume that MUI Core === `@mui/core`. But no, MUI Core = `@mui/core` + `@mui/system` + ` @mui/material` + `@mui/joy`

Reply:
I'm updating the content to use unstyled, for now, to keep the previous terminology